runtime/doc/api.txt

```diff
@@ -58,15 +58,15 @@ Nvim instance:
     # trailing '&' which is required since Nvim won't process events while
     # running a blocking command):
     #
-    #	:!./hello.rb &
+    #   :!./hello.rb &
     #
-    # Or from another shell by setting NVIM_LISTEN_ADDRESS:
-    # $ NVIM_LISTEN_ADDRESS=[address] ./hello.rb
+    # Or from another shell by setting NVIM:
+    # $ NVIM=[address] ./hello.rb
```

Comment:
NVIM as an environment variable is less descriptive than the original NVIM_LISTEN_ADDRESS in my humble opinion :).

Reply:
It doesn't need to be descriptive. Similar to `$TMUX`, it's mostly an implementation detail and otherwise revealed by `:help $NVIM`.

Reply:
> The usability afforded by terseness
Just like in 1972